configuration/nginx_http.conf

```diff
@@ -3,33 +3,39 @@ upstream django {
     server uwsgi:8001 fail_timeout=30s;
 }
 
-
 server {
     listen 80;
-
     server_name intel_owl.com;
 
     server_tokens off;
 
-    location = /favicon.ico { access_log off; log_not_found off; }
-
-    location /static/ {
-        alias /var/www/static/;
-   }
+    location = /favicon.ico { 
+        access_log off;
+        log_not_found off; 
+    }
 
-   location ~^/(admin|api) {
+    # Finally, send all non-media requests to the Django server.
+    location ~^/(admin|api|silk) {
```

Comment:
again out of curiosity, `silk` is for?

Reply:
The `silk/` endpoint is where the interface lives for the [django-silk](https://github.com/jazzband/django-silk) app. I intend to add this in IntelOwl as well so we can analyse performance and optimize. (#457).

Reply:
ah yes, this is the project you talked me about. 👍🏻